Images with caption

mahesh96s opened the pull request:
HTML page is designed with image and caption

mahesh96s committed:
Images with caption

mahesh96s committed:
Add files via upload

CSS is added in the html file

mahesh96s committed:
Removed box shadow in the images

manoj-v2 commented:
Align the code properly.

manoj-v2 commented on images.html:
```diff
@@ -0,0 +1,68 @@
+          <figure>
+            <img src="https://www.telegraph.co.uk/content/dam/Travel/Tours/India-xlarge.jpg"  >
+            <figcaption>India</figcaption>
+          </figure>
```
Since this is a list of images with caption, you should have embedded these figure tags inside ul>li

mahesh96s committed:
<ul> tag is added

Alignment of the code is done and then figure tags is embedded inside the list.

mahesh96s committed:
screenshot of the images

Images are aligned based on screenshot and then images content are linked to original web page

mahesh96s committed:
font weight is add to the links

manoj-v2 commented on screenshots.html:
```diff
@@ -0,0 +1,114 @@
+                    margin-left: -75px;
+                    margin-top:-10px;
+                    top: 0;
+                    margin-right:60px;
```
last_content and content have almost the same style definition exception for border-right.   Violates DRY princple -https://code.tutsplus.com/tutorials/3-key-software-principles-you-must-understand--net-25161

You can always use the last-child pseudo class to remove the border.

manoj-v2 replied:
https://google.github.io/styleguide/htmlcssguide.html .  Follow this guidelines when writing html/css

mahesh96s committed:
I have removed selector

manoj-v2 commented on screenshots.html:
```diff
@@ -0,0 +1,103 @@
+                <figure>
+                      <img src="https://i.gadgets360cdn.com/large/Poco-F1RThumb_1535476726411.jpg?downsize=130:98&output-quality=70&output-format=webp" >
+                      <figcaption class="date">Updated August 29,2018 </figcaption>
+                      <figcaption class="caption"><a href="https://gadgets.ndtv.com/mobiles/reviews/xiaomi-poco-f1-review-1907071?News_Trending">Xiaomi Poco F1 Review</a></figcaption>
```
Ideally only one image caption will be there for an image.

manoj-v2 commented on screenshots.html:
```diff
@@ -0,0 +1,103 @@
+                    margin-left: -65px;
+                    margin-top:-10px;
+                    top: 0;
+                    margin-right:60px;
```
combine all margin property into one.

manoj-v2 reviewed (commented):
Align the code properly.  Move the styles into an external file.

mahesh96s committed:
external css is added

1).I have merged all margin and padding and then external css is added.
2).I have removed <figcaption> tag.

mahesh96s committed:
save life
